zkevm-circuits/src/evm_circuit/step.rs

```diff
@@ -189,7 +190,7 @@ impl From<&ExecError> for ExecutionState {
                 }
                 OogError::MemoryCopy => ExecutionState::ErrorOutOfGasMemoryCopy,
                 OogError::AccountAccess => ExecutionState::ErrorOutOfGasAccountAccess,
-                OogError::CodeStore => ExecutionState::ErrorOutOfGasCodeStore,
+                OogError::CodeStore => ExecutionState::ErrorCodeStore,
```

Comment:
I guess this one is not necessary bcs `CodeStoreOutOfGas` was handled by L180.

Reply:
if no this line, will met build error  `&OogError::CodeStore` not covered